castle/cms/tiles/twittertimeline.py

```diff
@@ -11,40 +11,21 @@
 
 class TwitterTimelineTile(BaseTile):
     def render(self):
-        screenName = self.data.get('screenName', '')
+        self.screenName = self.data.get('screenName', '')
```

Comment:
`self.screenName`?

Doesn't seem to be assigning instance members on render.

If you need these in the template, you can do `view/data/screenName`